EnglishFilter/sections/foreign.txt

```diff
@@ -8996,6 +8996,11 @@ mm-cg.com##.TW_UTtoy
 ! https://github.com/AdguardTeam/AdguardFilters/issues/73710
 series-d.com##div[onclick^="window.open("]
 series-d.com##.skip-btn
+! https://github.com/AdguardTeam/AdguardFilters/issues/80747
+||cdend.com//*/*.mp4$domain=dunplayer.bid
+||dunplayer.download/*.mp4$domain=dunplayer.bid
+||dodonung.com/wp-content/plugins/better-adsmanager/js/advertising.min.js
+dodonung.com##div[style="margin: 8px 0; clear: both;"]
```

Comment:
Why not to block banners by url?
`||bp.blogspot.com^$image,domain=dodonung.com`